modules/xfmg/oas/XMOM/xmcp/oas/fman/ApplicationImport.xml

```diff
@@ -1,51 +1,66 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<!--
- * - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
```

Comment:
why is the copyright not here any more?

Reply:
Copyright Header is resolved with new commit.